.gitignore

```diff
@@ -6,20 +6,26 @@
 /test/*.csv
 /test/*.err
 /test/*.spl
+/test/*.bat
+/test/Testing/
```

Comment:
This shouldn't be here at all. A `Testing` directory indicates that you ran `ctest` from the wrong directory. It shouldn't be ignored because it should be obvious when someone makes this mistake, and they should delete the resulting directory.